zerver/openapi/zulip.yaml

```diff
@@ -3961,10 +3961,13 @@ paths:
                                         **Changes**: New in Zulip 5.0 (feature level 103).
                                     default_code_block_language:
                                       type: string
-                                      nullable: true
                                       description: |
-                                        The default pygments language code to be used for a code blocks
-                                        in this organization. If `null`, no default has been set.
+                                        The default pygments language code to be used for code blocks in this
+                                        organization. If an empty string, no default has been set.
+
+                                        **Changes**: Prior to Zulip 8.0 (feature level 193),`null` was documented
+                                        as the value that indicated no default had been set, when it was in fact
+                                        an empty string that did (and does) indicate this state.
```

Comment:
These events would have always had an empty string for the updated value when the default was unset, so that's what this **Changes** note should describe. Not 100% on "this state".

Reply:
I think what we want is something like "Prior to Zulip 8.0 (feature level 193), a bug meant that both `null` and `""` could represent no default language in `POST /register`, and the documentation for both endpoints incorrectly stated that `null` was the only representation for no default language. This event in fact uses the empty string to indicate no default language in all versions."